liquibase-core/src/main/java/liquibase/GlobalConfiguration.java

```diff
@@ -215,6 +216,11 @@ public class GlobalConfiguration implements AutoloadedConfigurations {
         ALWAYS_DROP_INSTEAD_OF_REPLACE = builder.define("alwaysDropInsteadOfReplace", Boolean.class)
                 .setDescription("If true, drop and recreate a view instead of replacing it.")
                 .build();
+
+        VALIDATE_XML_CHANGELOG_FILES = builder.define("validateXmlChangelogFiles", Boolean.class)
```

Comment:
@filipelautert Do we need to mention in the description about the performance overhead with enabling XSD validation?

Reply:
I"m addind some short comments about it.